src/System.Private.CoreLib/src/System/Reflection/RuntimeMethodInfo.cs

```diff
@@ -528,11 +531,11 @@ public override ParameterInfo ReturnParameter
                 return (m_returnParameter as ParameterInfo)!;
```

Comment:
Nit: This can be changed to just `return FetchReturnParameter();`

Reply:
Fixed.